### PR TITLE
{perf}[foss/2016a] Scalasca & Score-P update (REVIEW)

### PR DESCRIPTION
**Pull request by geimer**
Updated easyconfigs for the latest versions (released one hour ago) including deps.

Requires:
 - ~~https://github.com/hpcugent/easybuild-easyconfigs/pull/2908~~
 - https://github.com/hpcugent/easybuild-easyblocks/pull/889

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

@geimer: try to limit this PR to 10 easyconfigs, if you need more, it's better to open a separate PR for a subgraph of the dependency tree, to keep things manageable

**Comment by geimer** [comment]
@boegel: It'll be around 10, no worries.

**Comment by boegel** [comment]
@geimer OK, give me a sign if you feel it's complete.

**Comment by geimer** [comment]
Although the build probably doesn't fail, https://github.com/hpcugent/easybuild-easyblocks/pull/889 is needed to correctly enable all features.

**Comment by boegel** [comment]
@geimer seems like this also depends on #2908 (which just got merged)

Please mention all required PRs in the PR description (which you can edit). Bonus points to 'cross out' merged PRs using `~~...~~`.

**Comment by geimer** [comment]
@boegel: Yep, indeed.  Forgot about it when pushing the latest changes...

**Comment by geimer** [comment]
Test report by @geimer
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
swat - Linux debian 7.10, Intel(R) Xeon(R) CPU           X5550  @ 2.67GHz, Python 2.7.3
See https://gist.github.com/3554899530b1511fe62becc044108726 for a full test report.

**Comment by geimer** [comment]
Tested on top of https://github.com/hpcugent/easybuild-easyblocks/pull/889

File: easybuild/easyconfigs/s/Score-P/Score-P-2.0.1_fix_score_configure.patch [hunk 1]
```diff
@@ -0,0 +1,52 @@
+--- build-score/configure.orig	2016-05-11 09:21:03.716441529 +0200
```
Comment (boegel) [inline easybuild/easyconfigs/s/Score-P/Score-P-2.0.1_fix_score_configure.patch]: @geimer: please include a comment above this line to explain what this patch does, and mention yourself as an author

```
prevent build-score/configure from picking up $CFLAGS etc., backported from Score-P 2.0.2
author: Markus Geimer (JSC)
```

**Comment by boegel** [comment]
@geimer: I ran into this when testing this PR:

```
../src/.libs/libunwind-x86_64.so: error: undefined reference to 'lzma_stream_footer_decode' 
```

It seems like XZ needs to be added as a dependency...

**Comment by boegel** [comment]
@geimer please see https://github.com/geimer/easybuild-easyconfigs/pull/4, which handles the `XZ` issue (https://github.com/hpcugent/easybuild-easyconfigs/pull/3055 tackles the other existing `libunwind` easyconfigs)

**Comment by geimer** [comment]
Test report by @geimer
**SUCCESS**
Build succeeded for 9 out of 9 (8 easyconfigs in this PR)
swat - Linux debian 7.10, Intel(R) Xeon(R) CPU           X5550  @ 2.67GHz, Python 2.7.3
See https://gist.github.com/5d62265636d375897761c2ca09d65448 for a full test report.

**Comment by boegel** [comment]
going in, thanks @geimer!

**Pull request merged**